_posts/2016-05-10-omero-5-2-3.md

```diff
@@ -27,7 +27,7 @@ System administrator updates include:
 -  warnings added regarding the end of Windows support in the 5.3.0 release`[1]` (note that we will be preparing a guide for migrating from Windows for existing servers and adding it to the documentation as soon as we can)
 
 This release also upgrades the version of Bio-Formats which OMERO uses to
-5.1.9. For more information see https://www.openmicroscopy.org/site/support/bio-formats5.1/about/whats-new.html.
+5.1.9. For more information see [What's new]({{ site.baseurl }}/support/bio-formats5.1/about/whats-new/).
```

Comment:
this is a docs link and can't use the 'site.baseurl' - just keep the full old URL for now